modules/tinymce/src/themes/silver/test/ts/browser/editor/toolbar/NumberInputTest.ts

```diff
@@ -418,6 +420,31 @@ describe('browser.tinymce.themes.silver.throbber.NumberInputTest', () => {
     TinyAssertions.assertContent(editor, `<p style="font-size: ${originalFontSize};">a<span style="font-size: ${originalFontSize};">b</span>c</p>`);
   });
 
+  it('TINY-10330: the presence and updates of fontsizeinput should not trigger warnings', async () => {
+    // eslint-disable-next-line no-console
+    const storedConsoleWarn = console.warn;
+    const warnings: string[] = [];
+
+    // eslint-disable-next-line no-console
+    console.warn = (a) => {
+      // eslint-disable-next-line no-console
+      console.log('a: ', a);
```

Comment:
Maybe call `storedConsoleWarn` here if that is possible?

Reply:
thanks, I forget that `console.log` :sweat_smile: